src/responder/common/responder_common.c

```diff
@@ -1311,6 +1319,10 @@ int sss_process_init(TALLOC_CTX *mem_ctx,
         goto fail;
     }
 
+    sss_chain_id_setup(rctx->ev);
```

Comment:
Maybe it makes sense to move this call to `server_setup()` (and remove from `data_provider_be.c::main()`)? Just to have it in a single place.

Reply:
I tried this but it caused some issues with RID# tag missing on all applicable messages in the backend. I'll have to look into this further later if it is necessary.

Reply:
While not strictly necessary, would be good to at least understand this, imo.

Reply:
Any ideas?

Reply:
I made this change locally and it is working now, but I will wait for responses on the other review items before pushing, as it may be depend on the final solution.

Reply:
This is fixed and pushed